packages/elements-core/src/components/Docs/Docs.tsx

```diff
@@ -24,6 +24,16 @@ interface BaseDocsProps {
    * URI of the document
    */
   uri?: string;
+
+  /**
+   * Allows to hide TryIt component
+   */
+  hideTryIt?: boolean;
+
+  /**
+   * Shows only operation document without right column
+   */
+  hideTryItPanel?: boolean;
```

Comment:
Is this just for a direct usage by users? As of right now they won't be able to pass `hideTryItPanel` to the API component, just when they use `Docs`. What is the purpose for that?

Reply:
Yes, that's on purpose. We don't want to expose that property, rather keep it for internal usage